content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+```
+[1]: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-socket-option
+{{% /tab %}}
+
+{{% tab "ECS" %}}
+
+To achieve a single point of configuration across all telemetry, make `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` available within your container's runtime environment. These variables can be set in the Docker image itself or in the ECS task's definition:
+
+```yaml
+        "environment": [
+          {
+            "name": "DD_ENV",
+            "value": "<ENV>"
+          },
+          {
+            "name": "DD_SERVICE",
+            "value": "<SERVICE>"
+          },
+          {
+            "name": "DD_VERSION",
+            "value": "<VERSION>"
+          }
+        ]
+```
+
+If there are containers that will not benefit from using the environment variables in the application runtime, you can instead use Docker labels:
+
+```yaml
+        "dockerLabels": {
+          "com.datadoghq.tags.env": "<ENV>",
+          "com.datadoghq.tags.service": "<SERVICE>",
+          "com.datadoghq.tags.version": "<VERSION>"
+        }
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Non-containerized environment
+
+#### Setting variables
+Depending on how you build and deploy your services' binaries or executables, you may have several options available for setting `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. Since you may run one or more services per host, we recommend that these environment variables be scoped to a single process.
+
+You may elect export the environment variables in the command for your executable:
+
+`DD_ENV=<env> DD_SERVICE=<service> DD_VERSION=<version> /bin/my-service`
+
+Or you might use [Chef][12], [Ansible][13], or another orchestration tool to populate a service's systemd or initd configuration file with the `DD` environment variables. That way when the service process is started it will have access to those variables.
+
+#### Configuring traces, logs, and StatsD metrics
+
+{{< tabs >}}
+{{% tab "Traces" %}}
+
+As a best practice, it is recommended to configure your [APM tracer][1] with `DD_ENV` as it keeps the definition of `env` closer to the application that is generating the traces. This method allows the `env` tag to be sourced automatically from a tag in the span metadata.
+
+**Note: There can only be one service per span.** Trace metrics generally have a single service as well. However, if you have a different service defined in your hosts' tags, that configured service tag will show up on all trace metrics emitted from that host.
+
+[1]: /tracing/send_traces/
+{{% /tab %}}
+
+{{% tab "Logs" %}}
+
+If you already have [traces enabled for logs injection][1], it will also inject `env`, `service`, and `version` into your logs.
+
+If you already have logs configuration checks defined similarly to the example below, you **do not** need to define `service`:
+
+```yaml
+logs:
+  - type: file
+    path: "/path/to/your/log.log"
+    source: python
+    log_processing_rules:
+      ...
+    tags:
+      ...
```

Comment:
Don't we support the `init_config` section as well?

I would have hope in this section that we would have one example with `init config` working for all data types.

And then more details about how to override the value for logs and APM with injection from code to configure it without going through the agent.

**Why this structure?**

Mostly because there are two targets here:

1. Devops that have access to both the code and agent configuration. Setting everything up in the agent is possible and potentially the recommended way
2. Separated Dev | infrastructure teams where the deployment is handled by infra which do not have access to service, version information which then must be injected by developers directly.

Which is why I would say that we would need to explicitely call out the two setups.